trunk/src/app/srs_app_server.cpp

```diff
@@ -1227,11 +1226,26 @@ srs_error_t SrsServer::do_on_tcp_client(ISrsListener* listener, srs_netfd_t& stf
         }
     }
 
+    // For RTC TCP connection, use resource executor to manage the resource.
```

Comment:
It's better to add conditional compile by following above: `#ifdef SRS_RTC`.

Reply:
I notice this problem, when I check the github workflow test error on `build-centos7`.
To reproduce: `./configure -rtc=off; make`

To fix this compiling error, there are still some places to fix.
1. `src/app/srs_app_stream_bridge.cpp`, delete line 11, `// #include <srs_app_rtc_codec.hpp>`, because there is no ref of `SrsAudioTranscoder` in `srs_app_stream_bridge.cpp`;
2. `src/app/srs_app_stream_bridge.hpp`, there is a `SrsAudioTranscoder` forward declaration, it seems useless, there is no warning, but useless. (L22);
3. fix warning:
https://github.com/ossrs/srs/blob/6834ec208d67fa47c21536d1f1041bb6d60c1834/trunk/src/app/srs_app_threads.cpp#L271-L281
   move `buf` declaration into the `SRS_RTC` conditional compiling block.
4. fix warning:
https://github.com/ossrs/srs/blob/6834ec208d67fa47c21536d1f1041bb6d60c1834/trunk/src/app/srs_app_stream_bridge.hpp#L62-L63
    according to the `srs_app_stream_bridge.cpp`, the warning can be fixed by following conditional compiling.
    ```
    #ifdef SRS_RTC
    SrsRtcSource* source_;
    #endif
    #ifdef SRS_FFMPEG_FIT
    SrsRtcRtpBuilder* rtp_builder_;
    #endif
    ```
    But I would rather suggest reconsider the conditional compiling in `srs_app_stream_bridge.cpp`. e.g. 
    ```
    #if defined(SRS_RTC) && defined(SRS_FFMPEG_FIT)
    srs_freep(rtp_builder_);
    #endif
    ```